ext/standard/tests/strings/str_starts_with.phpt

```diff
@@ -0,0 +1,47 @@
+--TEST--
+str_starts_with() function - unit tests for str_starts_with()
+--FILE--
+<?php
+/* Prototype: bool str_starts_with (string $haystack, string $needle);
+   Description: Determine if $haystack begins with $needle
+*/
+$testStr = "beginningMiddleEnd";
+var_dump(str_starts_with($testStr, "beginning"));                                                                             
+var_dump(str_starts_with($testStr, "Beginning"));                                                                             
+var_dump(str_starts_with($testStr, "eginning"));  
+var_dump(str_starts_with($testStr, $testStr."a"));  
+var_dump(str_starts_with($testStr, ""));
+var_dump(str_starts_with("", ""));
+var_dump(str_starts_with("", " "));
+var_dump(str_starts_with($testStr, "\x00"));
+var_dump(str_starts_with("\x00", ""));
+var_dump(str_starts_with("\x00", "\x00"));
+var_dump(str_starts_with("\x00a", "\x00"));
+var_dump(str_starts_with("\x00a", "a"));
+var_dump(str_starts_with("\x00ab", "\x00a"));
+var_dump(str_starts_with("\x00ab", "\x00d"));
```

Comment:
I would replace the 3 tests above with a single `("a\x00bc", "a\x00b")` (expect `bool(true)`)